# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h
# Commit: 46af55564501
# Full Hash: 46af555645013a126416a440b658f2969eca4b20
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-06 08:52:03
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 55c4afa3a46c -r 46af55564501 toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h
--- a/toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h	Thu Jun 05 14:53:36 2025 +0000
+++ b/toolkit/components/credentialmanagement/IdentityCredentialRequestManager.h	Thu Jun 05 14:53:36 2025 +0000
@@ -25,7 +25,6 @@
   static StaticRefPtr<IdentityCredentialRequestManager> sSingleton;
   IdentityCredentialRequestManager() {};
   ~IdentityCredentialRequestManager() = default;
-
 };
 
 }  // namespace mozilla